tests/lib/rules/no-useless-return.js

```diff
@@ -418,11 +429,7 @@ ruleTester.run("no-useless-return", rule, {
         },
         {
             code: "function foo() { return; return; }",
-            output: "function foo() {  return; }", // Other case is fixed in the second pass.
-            errors: [
-                { message: "Unnecessary return statement.", type: "ReturnStatement" },
-                { message: "Unnecessary return statement.", type: "ReturnStatement" }
-            ]
+            output: "function foo() {  return; }"
```

Comment:
I think it would be good to have an `errors` object showing 1 error, and use the column option to show that the first `return` is the one flagged.